.storybook/recipes/GlobalHeader/GlobalHeader.module.css

```diff
@@ -49,17 +49,31 @@
   color: var(--eds-theme-color-text-neutral-default-inverse);
 
   &:hover,
-  &:focus {
+  &:focus-visible {
     background: var(
       --eds-theme-color-button-primary-brand-background-hover
     ); /* TODO */
     border-color: transparent; /* TODO */
   }
 
-  &:focus {
+  &:focus-visible {
     @mixin focusInverted;
   }
 
+  @supports not selector(:focus-visible) {
+    &:hover,
+    &:focus {
+      background: var(
+        --eds-theme-color-button-primary-brand-background-hover
+      ); /* TODO */
+      border-color: transparent; /* TODO */
+    }
```

Comment:
I'm duplicating the whole block, including the `hover` property, because otherwise the `hover` styles are ignored in browsers that don't support `focus-visible`.